action.yml

```
@@ -0,0 +1,135 @@
+name: Tag and release
+description: GitHub Action to create a tag and an optional release
+
+inputs:
+  # Note: there is an explicit reason why there is no sha input parameter - see the readme
+  tag:
+    type: string
+    required: true
+  delete_existing:
+    type: boolean
+    required: false
+    default: false
+  release:
+    type: boolean
+    required: false
+    default: false
+  body:
```

```suggestion
  release_body:
```
Makes it immediately clearer what this is for (as opposed to, say, a comment on an annotated tag)

---

Have changed to `release_description`